website/src/components/TaskSelection/TaskOption.tsx

```diff
@@ -12,6 +13,8 @@ export type OptionProps = {
 export const TaskOption = (props: OptionProps) => {
   const { alt, img, title, link } = props;
   return (
+    
+    <Tooltip label={label}>
```

Comment:
You didn't deconstruct the label props. Also, please add the type to the `OptionProps`.